modules/electron/system-part3/makefile

```diff
@@ -26,6 +26,8 @@ TARGET=elf bin lst hex size
 
 include $(PROJECT_ROOT)/build/arm-tlm.mk
 
+LDFLAGS += -flto -Os -fuse-linker-plugin -Wl,--undefined=vTaskSwitchContext
```

Comment:
A comment on `vTaskSwitchContext` would be nice here.

Reply:
Added.